### ISSUE TITLE
!stop emp handling

### ISSUE THREAD

**Issue by ImTheDeveloper** [d7415/merlin#12]
The current stop command does not take into account emp init

Stopping 1900 Vindicator (11.4k) as t1 requires Gryphon: 1131 (12.4k) Centaur: 699 (15k) Spider: 2639 (11.9k) Scarab: 660 (12.6k) Revenant: 1320 (11.8k) Ghost: 679 (12.1k) Cutlass: 1080 (11.3k) Vindicator: 1584 (9504) Ranger: 660 (13.1k)

In the example spider needs to be around ~5.6k due to being emp'd first by vindicator.

Unsure if its relevant for !eff since we do not account for ships dieing first there

**Comment by d7415** [comment]
> Unsure if its relevant for !eff since we do not account for ships dieing first there

I don't think we currently consider that in !stop either. I have a patch (not mine) that I was going to incorporate at some point and then apparently didn't. I'll look at adding it now.

Aside: It's getting towards time to look at switching to other, better maintained tools or someone forking this. [munin](https://github.com/munin/munin) has been getting much more attention recently.

**Comment by d7415** [comment]
Looks like the patch only handles norm <-> norm and emp <-> emp so will need more work...

**Comment by d7415** [comment]
I don't have time to fix it right now. If you want to have a go, I've posted it here: https://github.com/d7415/merlin/commit/e8014c1c041e1550458c874975c5d0abcc8ade1f. !stop doesn't seem to consider emp targets, !eff doesn't consider norm attacking emp or vice versa.
